sdk/formrecognizer/ai-form-recognizer/test/browser/formtrainingclient.spec.ts

```diff
@@ -151,7 +152,8 @@ describe("FormTrainingClient browser only", () => {
 
   it("listModels() iterates models in this account", async () => {
     let count = 0;
-    for await (const _model of trainingClient.listCustomModels()) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    for await (const _ of trainingClient.listCustomModels()) {
```

Comment:
Can we not refactor this to use the simpler `for` loop?
```js
for (let i = 0; i < 30 i++) {
     await trainingClient.listCustomModels();
}
```

Reply:
I am afraid that the proposed loop would be different as this is async iterator.  The related issue has been fixed now. So the counting and breaking at 30 are not needed any more.  We can assert on valid _model.id if it makes the linting issue to go away.

Reply:
ah interesting, so perhaps a while loop with calling next() on the iterator would be an alternative?
but yes, if we don't need to count to 30 anymore, that would be nice :)